app/controllers/email_alert_subscriptions_controller.rb

```diff
@@ -60,29 +62,27 @@ def fetch_content_item(content_item_path)
 
   def email_alert_signup_api
     EmailAlertSignupAPI.new(
-      email_alert_api: Services.email_alert_api,
-      attributes: email_signup_attributes,
-      default_attributes: { filter: default_filters, reject: default_rejects },
-      subscription_list_title_prefix: content['details']['subscription_list_title_prefix'],
-      available_choices: signup_presenter.choices,
+      applied_filters: applied_filters,
```

Comment:
I like the refactor here, but I'm thinking it might be a good idea (at some point) to move all logic out of this controller and into the EmailAlertSignupAPI.

Reply:
It might also be worth considering `email-alert-frontend`. Email signup logic will probably grow in complexity so sticking it all in one app away from search / finders might be worth it.